tests/util/kubeUtils.go

```diff
@@ -225,15 +225,20 @@ func GetPodsName(n string) (pods []string) {
 
 // GetPodStatus gets status of a pod from a namespace
 func GetPodStatus(n, pod string) string {
-	status, err := Shell("kubectl -n %s get pods %s -o jsonpath='{.status.phase}'", n, pod)
+	status, err := Shell("kubectl -n %s get pods %s --no-headers", n, pod)
 	if err != nil {
 		log.Infof("Failed to get status of pod %s in namespace %s: %s", pod, n, err)
 		status = podFailedGet
 	}
-	return strings.Trim(status, "'")
+	f := strings.Fields(status)
+	if len(f) > 2 {
```

Comment:
Try define the index constant 2 with a descriptive name instead of hard coding it.

Reply:
Done